src/routes/users/registerLogin.js

```diff
@@ -69,36 +63,25 @@ const apiLimiter = new RateLimit({
 
 router.use(loginPlatforms);
 
-function csrfCheck(req, res, next) {
-  const { 'user-agent': userAgent = '' } = req.headers;
-  if (userAgent.includes('LikeCoinApp')) {
-    next();
-  } else {
-    csrf({ cookie: CSRF_COOKIE_OPTION })(req, res, next);
-  }
-}
-
-// deprecated
 function isJson(req) {
   return !!req.is('application/json');
 }
 
-// deprecated
-function loadMiddlewareByContentType(req, res, next) {
+function isApp(req) {
+  const { 'user-agent': userAgent = '' } = req.headers;
+  return userAgent.includes('LikeCoinApp');
+}
+
+function formdataParserForApp(req, res, next) {
   if (!isJson(req)) {
-    csrfCheck(req, res, (csrfErr) => {
-      if (csrfErr) next(csrfErr);
-      bodyParser.urlencoded({ extended: false })(req, res, (bpErr) => {
-        if (bpErr) next(bpErr);
-        multer.single('avatarFile')(req, res, next);
-      });
-    });
+    if (isApp(req)) multer.none()(req, res, next);
+    else throw new ValidationError('INVALID_CONTENT_TYPE');
```

Comment:
Error show be called as a param in `next()` in express middleware, i.e.
```suggestion
    else next(new ValidationError('INVALID_CONTENT_TYPE'));
```